exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -252,3 +268,35 @@ func attrValue(value pcommon.Value) interface{} {
 		return nil
 	}
 }
+
+func (e *exporter) getPusher(logGroup, logStream string) cwlogs.Pusher {
```

Comment:
Is this the same logic from the EMF exporter? Could/should it move into the `cwlogs` package?

Reply:
I thought about it, but there we don't want to actually share the instance between the two.  Each processor should have its own scope.  We could make a new interface that does this, but there are some things that might  bite us.

If you are really keen on this, I'll do it.  For now I am content to see how this plays out before refactoring anymore.

Let me know @Aneurysm9

Reply:
I wouldn't suggest we share the same instance between the two, but if the logic for having a pool of pushers is common then it would perhaps make sense to move that to the common package.

Reply:
Yeah I just fear their APIs might converge at some point forcing them to be refactored out.